packages/wallets/src/AppContent.tsx

```diff
@@ -6,9 +6,11 @@ import './AppContent.scss';
 const AppContent: React.FC = () => {
     const { is_mobile } = useDevice();
 
+    if (is_mobile) return <WalletsCarousel />;
+
     return (
         <div className='wallets-app'>
-            <div className='wallets-app__content'>{is_mobile ? <WalletsCarousel /> : <DesktopWalletsList />}</div>
```

Comment:
Shouldnt we keep it like this instead?

Reply:
The styles are different for the parent in mobile and desktop
We go for now.